### ISSUE TITLE
Segment fewer numbers of object categories

### ISSUE THREAD

**Issue by sronast** [facebookresearch/Mask2Former#102]
Using a pre-trained model in coco, is it possible to get segmentation for only 2 object categories(eg. cat and dog)?

**Comment by bowenc0221** [comment]
Yes, you can do this by keeping the prediction from two categories (out of 80 COCO instance categories).

**Comment by sronast** [comment]
Should I need to filter the output in predict.py file? or is there an elegant way to do it?
